src/exabgp/bgp/message/update/nlri/mup/t1st.py

```diff
@@ -165,8 +185,24 @@ def unpack(cls, data, afi):
         size += 1
         endpoint_ip_len = data[size]
         size += 1
-        endpoint_ip = IP.unpack(data[size:])
-        return cls(rd, ipprefix_len, ipprefix, teid, qfi, endpoint_ip_len, endpoint_ip, afi)
+
+        if endpoint_ip_len in [32, 128]:
+            endpoint_ip = IP.unpack(data[size : size + endpoint_ip_len])
+            size += endpoint_ip_len
+        else:
+            raise RuntimeError('endpoint ip length is not 4 or 16')
```

Comment:
can the runtime error be a bit more specific about which endpoint ip is wrong to help people when they look at the logs.